package.json

```diff
@@ -6,24 +6,22 @@
     "packages/*"
   ],
   "dependencies": {
-    "husky": "8.0.1",
-    "lerna": "5.5.0"
+    "husky": "8.0.1"
   },
   "devDependencies": {
     "lerna-update-wizard": "1.1.0"
```

Comment:
I think we may be able to remove this? I don't think renovate relies on it to do its thing

Reply:
I don't think so... apparently it's used by running `lernaupdate` and it comes up with a window in the terminal requiring user input and stuff but I can't see this command being run anywhere at any point and I don't believe renovate uses it either. Just to double check though, we'll quickly confirm with @louise-davies if she doesn't mind chiming in here?

Reply:
Yeah, `lerna-update-wizard` can be removed - I used to use it before we had automated dependency updates

Reply:
Okay cool, thanks.